src/commands/install/utils/verify_repo_url.py

```diff
@@ -0,0 +1,5 @@
+import re
+
+
+def verify_repo_url(url: str) -> bool:
+    return bool(re.match(r"^https:\/\/github.com\/[^\/]*\/[^\/]*$", url))
```

Comment:
would ssh clones be possible?

Reply:
I am not sure what you mean by `ssh clones`. Can you provide an example?